src/com/oltpbenchmark/WorkloadConfiguration.java

```diff
@@ -396,13 +332,6 @@ public void setShouldEnableForeignKeys(boolean shouldEnableForeignKeys) {
     this.shouldEnableForeignKeys = shouldEnableForeignKeys;
   }
 
-  public boolean getCreateSQLProcedures() {
```

Comment:
is this not used?

Reply:
correct

https://github.com/yugabyte/tpcc/search?q=getCreateSQLProcedures